### PR TITLE
2632: add total over orders to dashboard and fix other totals labels

### PR DESCRIPTION
**Pull request by ltello**
### Context

[Card](https://trello.com/c/WObDY3mf)

### CODE REVIEW

File: app/models/support/service_performance.rb [hunk 1]
```diff
@@ -33,40 +33,56 @@ def number_of_centrally_managed_schools
...
+  def total_devices_allocated
+    total_devices_cap - total_devices_over_ordered
+  end
+
+  def total_devices_cap
     sum_allocation(Arel.sql("CASE order_state WHEN 'cannot_order'
                                         THEN raw_laptops_ordered
                                         ELSE raw_laptop_allocation + over_order_reclaimed_laptops + circumstances_laptops
                                     END"))
   end
```
Comment (markhallen) [inline app/models/support/service_performance.rb:45]: Maybe we should memoise this as it it is called twice?

File: app/models/support/service_performance.rb [hunk 2]
```diff
@@ -33,40 +33,56 @@ def number_of_centrally_managed_schools
...
+  def total_routers_allocated
+    total_routers_cap - total_routers_over_ordered
+  end
+
+  def total_routers_cap
+    sum_allocation(Arel.sql("CASE order_state WHEN 'cannot_order'
+                                        THEN raw_routers_ordered
                                         ELSE raw_router_allocation + over_order_reclaimed_routers + circumstances_routers
                                     END"))
   end
```
Comment (markhallen) [inline app/models/support/service_performance.rb:71]: Could be worth memosiing this too.

File: config/locales/en.yml [hunk 3]
```diff
@@ -403,9 +403,10 @@ en:
         one: responsible body has given all the information needed for at least one school
         other: responsible bodies have given all the information needed for at least one school
     devices_performance:
-      total_devices_available: total devices available
-      total_devices_ordered: devices shipped
-      total_devices_remaining: remaining
+      total_devices_allocated: total devices allocated
+      total_devices_ordered: devices ordered
```
Comment (markhallen) [inline config/locales/en.yml:407]: 👏 the old text made no sense at all

**Review by markhallen — APPROVED** [review]

**Review by markhallen — APPROVED** [review]

File: app/models/support/service_performance.rb (hunk 1 above)
Comment (ltello) [reply app/models/support/service_performance.rb]: Sure, all of these totals should be memoized

**Comment by ltello** [comment]
[Service allocation liability feature](https://trello.com/c/WObDY3mf/2632-service-allocation-liability-feature)

**Pull request merged**